services/ui-src/src/utils/autosave/autosave.ts

```diff
@@ -26,56 +24,70 @@ interface Props {
     userName: string | undefined;
     state: string | undefined;
   };
+  fieldType: string;
```

Comment:
Warning: Passing through fieldType here will break your implementation anytime you're passing through more than one field for autosave at a time (i.e. when nested child fields are being blanked). That's why we have all field-specific data in the `FieldInfo` type object that comes through. If we decide we need to pass field type through, we should do it for each field that's being saved.